ansible/roles/setup_host/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+- name: install packages
+  become: yes
+  yum:
+    name:
+    - docker
+    - git
+    state: present
+  tags:
+    - packages
+
+- name: add docker group
+  become: yes
+  group:
+    name: docker
+    state: present
+
+- name: add user to docker group
+  become: yes
+  user:
+    name: "{{ user }}"
+    groups: docker
+    append: yes
+
+- name: start docker
+  become: yes
+  systemd:
+    name: docker
+    enabled: yes
+    state: started
+
+- include: download.yml
```

Comment:
No! still bad! install.yml doesn't use download.yml, why would it include it? ..

Reply:
As discussed, it's part of the host preparation at the moment, we can remove it later if needed